src/main/java/com/gregtechceu/gtceu/api/machine/trait/RecipeHandlerGroup.java

```diff
@@ -0,0 +1,4 @@
+package com.gregtechceu.gtceu.api.machine.trait;
+
+public interface RecipeHandlerGroup {
+}

```

Comment:
we want to force people to implement equals() and hashCode() here, also, a comment would be helpful.
```suggestion
public interface RecipeHandlerGroup {

    @Override
    boolean equals(Object o);

    @Override
    int hashCode(Object o);
}

```

Reply:
hashcode doesn't take an object, so removed that, apart from that implemented